pyfr/backends/openmp/xsmm.py

```diff
@@ -46,7 +46,7 @@ def __del__(self):
 
             self._wrappers.libxsmm_finalize()
 
-    def mul(self, a, b, out, alpha=1.0, beta=0.0):
+    def mul(self, a, b, out, alpha=1.0, beta=0.0, a_facs=[]):
```

Comment:
Declare a new method for chained multiplications.  Have other backends just pre-multiply the factors and defer to mul.